lib/v2/nua/gra.js

```diff
@@ -0,0 +1,60 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const baseUrl = 'https://grad.nua.edu.cn/';
+
+module.exports = async (ctx) => {
+    const type = ctx.params.type;
+    const newsUrl = `${baseUrl}/${type}/list.htm`;
+
+    const response = await got({
+        method: 'get',
+        url: newsUrl,
+        https: {
+            rejectUnauthorized: false,
+        },
+    });
```

Comment:
got request can be simplified to 
```suggestion
    const response = await got(newsUrl, {
        https: {
            rejectUnauthorized: false,
        },
    });
```